Magento2/Internal/FileProxy.php

```diff
@@ -28,6 +28,7 @@ public function __construct(\PHP_CodeSniffer\Files\File $file, array $proxies =
     {
         $this->file = $file;
         $this->proxies = $proxies;
+        $this->fixer = $file->fixer;
```

Comment:
Please use magic methods `_get()/_set()` in oder to fix the issue for all public properties.

Reply:
This file is being removed in https://github.com/magento/magento-coding-standard/pull/452 (which I expect to be merged in this week), making further changes to this file seems like unnecessary work.

This change is the minimum required to make the tests pass. If you would like to add additional tests and write the necessary code to make them pass, please open a new pull request with those changes.

My recommendation would be to get this merged, then https://github.com/magento/magento-coding-standard/pull/453, then https://github.com/magento/magento-coding-standard/pull/452.